app/api.py

```diff
@@ -17,11 +18,19 @@ def fetch_from_api():
     """)
     url = 'https://parseapi.back4app.com/classes/Carmodels_Car_Model_List?limit=10&keys=Make,Year&where=%s' % where
     headers = {
-        'X-Parse-Application-Id': '1Fq1JQ9T6jfkbcGGOuzGVwMeOaLe6ArsmPKPXvfi',  # This is your app's application id
-        'X-Parse-REST-API-Key': '9gA1WmxS3RolZChj4tgHkHj3Firlxt6Yn16DsTEi'  # This is your app's REST API key
+        # This is your app's application id
+        'X-Parse-Application-Id': '1Fq1JQ9T6jfkbcGGOuzGVwMeOaLe6ArsmPKPXvfi',
+        # This is your app's REST API key
+        'X-Parse-REST-API-Key': '9gA1WmxS3RolZChj4tgHkHj3Firlxt6Yn16DsTEi'
     }
-    data = json.loads(
-        requests.get(url, headers=headers).content.decode('utf-8'))  # Here you have the data that you need
+
+    try:
+        data = json.loads(
+            requests.get(url, headers=headers).content.decode('utf-8'))  # Here you have the data that you need
+    except requests.exceptions.HTTPError as err:
+        logging.critical('Bad Request Code')

```

Comment:
try to look into how HTTPErrors and their response codes are handled efficiently. and if the request fails due to some reason, also mention the reason.